packages/editor-ui/src/components/RunDataJson.vue

```diff
@@ -229,11 +237,10 @@ export default defineComponent({
 	left: 0;
 	padding-left: var(--spacing-s);
 	right: 0;
-	overflow-y: auto;
+	overflow-y: hidden;
 	line-height: 1.5;
 	word-break: normal;
 	height: 100%;
-	padding-bottom: var(--spacing-3xl);
```

Comment:
Why are we getting rid of this? I think it looks worse without it (maybe just reduce it a bit since `3xl` seems too much to me)